src/twisted/conch/ssh/keys.py

```diff
@@ -624,6 +710,39 @@ def _fromDSAComponents(cls, y, p, q, g, x=None):
         return cls(keyObject)
 
 
+    @classmethod
+    def _fromECComponents(cls, x, y, curve, private_value=None):
```

Comment:
Twisted coding standard says that this variable should be called `privateValue`, not `private_value`.